src/Clap-Tests/ClapCodeEvaluatorTest.class.st

```diff
@@ -0,0 +1,38 @@
+Class {
+	#name : #ClapCodeEvaluatorTest,
+	#superclass : #ClapPharoCommandsTest,
+	#category : #'Clap-Tests-Commands'
+}
+
+{ #category : #tests }
+ClapCodeEvaluatorTest >> testEvaluateAliasMultipleArguments [
+	context := ClapCodeEvaluator evaluate activateWith: #('eval'
+		'''(2+3)''' 'class' 'printString' ', String space ,' '(2+3)' 'class' 'printString').
+	
+	self assertSuccess.
+	self
+		assert: self outputString
+		equals: '''ByteString SmallInteger''' , self lineEnding
+]
+
+{ #category : #tests }
+ClapCodeEvaluatorTest >> testEvaluateAliasSingleArgument [
+	context := ClapCodeEvaluator evaluate activateWith: #('eval' '1 class').
+	
+	self assertSuccess.
+	self
+		assert: self outputString
+		equals: 'SmallInteger' , self lineEnding
+]
+
+{ #category : #tests }
+ClapCodeEvaluatorTest >> testEvaluatePipeIn [
+	context := (ClapCodeEvaluator evaluate activationWith: #('evaluate'))
+		setStdio: (ClapPluggableStdio onByteArraysWithInput: 'Object class class class class');
+		execute.
+	
+	"self assertSuccess."
```

Comment:
Is this a leftOver?

Reply:
This should be removed.

Reply:
The other tests do that assertion (that the exit status is 0); I uncommented it